src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -117,8 +118,17 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
      */
     public AddressBook load() throws StorageOperationException {
+        File fileOnDisk = path.toFile();
+
+        // create empty file if not found
+        if (!fileOnDisk.exists()) {
```

Comment:
It is not sufficient to check if the specified file exists or not. What if I create a new folder called "addressbook.xml"?